supporting_scripts/analysis-of-endpoint-connections/src/main/java/analysisOfEndpointConnections/AnalysisOfEndpointConnections.java

```diff
@@ -0,0 +1,75 @@
+package analysisOfEndpointConnections;
```

Comment:
this is a weird package name

Reply:
Would you prefer a different package name?

Reply:
I would argue that package names should be lower case and consist of only one word separated by dots

Reply:
I renamed the Package.